Codecademy/gamut
# feat(Icons): MoneyBackGuarantee

### Overview


This adds the MoneyBackGuarantee Icon used currently in Brand. 
Also removes the fill and normalizes it to "none" like with the other icons.
Subsequent PR to remove the icon from Brand.


### PR Checklist

- [x] Related to JIRA ticket: [GM-895](https://skillsoftdev.atlassian.net/browse/GM-895)

#### Testing Instructions



1. Go to ?path=/story/atoms-icons-regular--editable-icons&args=iconName:MoneyBackGuaranteeIcon
2. At the top, select Canvas
3. In the `iconName` field, go to Control row and select `MoneyBackGuaranteeIcon` 
4. Check that it loads properly on both light and dark modes
5. Go check monolith's preview + mono's preview
7. ...
8. Profit!

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/38755)  | [Monolith Env](https://pr-38755-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/8354)  | [Portal Env](https://3603755d.brand-storybook.pages.dev/)   





[GM-895]: https://skillsoftdev.atlassian.net/browse/GM-895?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
